src/Microsoft.AspNet.SignalR.SqlServer/SqlMessageBus.cs

```diff
@@ -134,22 +134,23 @@ private void Initialize(object state)
             }
         }
 
-        private void StartReceiving(int streamIndex)
+        private async void StartReceiving(int streamIndex)
```

Comment:
@Xiaohongt  You can always just store the task in a variable and ignore it in the calling function. Since we are not going to observe the Task though, I would put `var stream = _streams[streamIndex];` in the try block just to be overly cautious.